API.Models/PlaceModels/PlaceDetail.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace API.Models
+{
+    public class PlaceDetail
+    {
+        [Key]
+        public int ID { get; set; }
+        [Required]
+        public string Name { get; set; }
+        [Required]
+        public string Position { get; set; }
+        public int Elevation { get; set; }
+        public string Climate { get; set; }
```

Comment:
Same issue as on the PlaceCreate model. [Key] and [Required] aren't needed here
```suggestion
        public int ID { get; set; }
        public string Name { get; set; }
        public string Position { get; set; }
        public int Elevation { get; set; }
        public string Climate { get; set; }
```